src/Parser.cpp

```diff
@@ -20,6 +20,7 @@
 #include <iostream>
 #include <optional>
 #include <sstream>
+#include <utility>
```

Comment:
Confirm this is needed (I forgot why I added this)?

Reply:
Compiles without this include. However, according to cppreference, this include provides, among other things, std::move. Keeping this include for this reason in case a different platform needs this.